code/modules/surgery/bodyparts/head.dm

```diff
@@ -257,6 +257,17 @@
 			if(eyes.eye_color)
 				eyes_overlay.color = eyes.eye_color
 
+/obj/item/bodypart/head/talk_into(mob/holder, message, channel, spans, datum/language/language, list/message_mods)
+	var/mob/headholder = holder
+	if(istype(headholder))
+		headholder.log_talk(message, LOG_SAY, tag = "beheaded talk")
+
+	say(message, language, sanitize = FALSE)
+	return NOPASS
+
+/obj/item/bodypart/head/GetVoice()
+	return "The head of [real_name]"
+
```

Comment:
this is here because otherwise you can behead a person to speak with their voice over radio, with the only drawback being the speech done in the robot text. while it's painfully obvious it's kinda not what I was going for. plus it's ventriloquism not necromancy